### PR TITLE
custom page header

### PR DESCRIPTION
**Pull request by derpy-duck**

### CODE REVIEW

File: wormhole-connect/src/config/index.ts [hunk 1]
```diff
@@ -31,6 +31,7 @@ const getEnv = () => {
 export const ENV = getEnv();
 export const isMainnet = ENV === 'MAINNET';
 export const sdkConfig = WormholeContext.getConfig(ENV);
+export const pageHeader = config.pageHeader || 'Bridge';
```
Comment (evan-gray) [inline wormhole-connect/src/config/index.ts]: 🤔 should we allow integrators to remove the page header?

**Review by evan-gray — COMMENTED** [review]
Reminder after this is in a release to update the builder.

File: wormhole-connect/src/config/index.ts (hunk 1 above)
Comment (evan-gray) [reply wormhole-connect/src/config/index.ts]: e.g. 
```
export const pageHeader = config.pageHeader === undefined ? 'Bridge' :  config.pageHeader;
```

Comment (M-Picco) [reply wormhole-connect/src/config/index.ts]: Maybe leave it as undefined by default and render it only if it is set?

Comment (derpy-duck) [reply wormhole-connect/src/config/index.ts]: Just allowed the empty string - I think undefined as default would modify behavior for existing integrators right (perhaps they want 'Bridge' as the title?)

Comment (derpy-duck) [reply wormhole-connect/src/config/index.ts]: JK - now the default is the empty string

Comment (evan-gray) [reply wormhole-connect/src/config/index.ts]: Agree this is a reasonable default change - not sure anyone _wanted_ "Bridge" but they can set it if they particularly enjoyed it, I guess.

**Review by evan-gray — APPROVED** [review]

**Pull request merged**